### DISCUSSION TITLE
Web page is showing just a symbol OI

### DISCUSSION THREAD

**Discussion by Noorulhuda90** [open-webui/open-webui#4453] [Q&A]
Hi,

When I type http://localhost:3000/ in the browser only a symbol OI appears. Otherwise the page is empty. what could be the reaseon ?

**Comment by zyxchaos0314** [comment]
the same, does this problem sovled?

**Comment by Noorulhuda90** [comment]
Hi, It was not an issue of web ui. It is an interface problem. You can try to change web browser and see on which localhost you are running the webui and try that instead of localhost. If you can provide me a little more details about your system and how are you accessing the server I may provide some more detailed answer.

**Comment by jwalcek** [comment]
Try a hard refresh (Control + F5).

**Comment by prkznq** [comment]
I've had the same problem, it was the browser version. Can you check with different browser or updated versions?
